### PR TITLE
adding examples, fyi, projects for kotlin

### PR DESCRIPTION
**Pull request by colleend**
Notes:
* changed scripts/lang-stat because `wc -l <X> | cd -d ' ' -f1` command led to empty line count.
* on OSX, grep -P does not exist because the version installed automatically is from FreeBSD and doesn't have the -P flag. Included in the instructions how to bypass this, but we can change grep -P to something else later.

- [x] Added examples for kotlin.
- [x] Added extensions.txt with kotlin extensions.
- [x] Updated adding-a-language doc.
- [x] Created scripts/most-starred-for-language in order to grab most popular github repos for a specific language.
- [x] Added repos to projects.txt
- [x] Updated to new version of semgrep-grammars

### CODE REVIEW

File: scripts/most-starred-for-language.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    s.headers.update({'Accept': 'application/vnd.github.cloak-preview+json'})
+
+    url = 'https://api.github.com/search/repositories?q='+ args.lang + '&s=stars&type=Repositories'
+    r = s.get(url)
+    pretty_json = json.loads(r.text)
+
+    with open('projects.txt', 'w') as f:
+        for repo in pretty_json["items"]:
+            f.write(repo["html_url"]+"\n")
+
```
Comment (mjambon) [inline scripts/most-starred-for-language.py:33]: oh, nice!

**Review by mjambon — APPROVED** [review]
awesome

File: scripts/most-starred-for-language.py (hunk 1 above)
Comment (aryx) [reply scripts/most-starred-for-language.py:33]: nice one indeed!

**Review by aryx — COMMENTED** [review]
Great!

**Pull request merged**